drivers/telescope/lx200telescope.cpp

```diff
@@ -640,7 +640,7 @@ bool LX200Telescope::updateLocation(double latitude, double longitude, double el
     if (isSimulation())
         return true;
 
-    if (!isSimulation() && setSiteLongitude(PortFD, 360.0 - longitude) < 0)
+    if (!isSimulation() && setSiteLongitude(PortFD, longitude) < 0)
```

Comment:
Is 360.0 - longitude the same as flipping the sign?

Reply:
this `360 -` actually caused problems for me, my longitude is -5 , and then this results in 365 degrees which my mount refuses to set. There were more places with longitude 'calculations' and these became only visible when I upgraded the logging to always log both sexagesimal and the double. The visual display in KStars uses carthographic coordinates because that is what users read on maps and get from GPS, INDI internally converted that halfway to 0-360 which is actually not needed for the api in the first place. The new standard is carthographic everywhere so that my brain does not hurt, and convert for lx200 during actual writing to and reading from the mount. This conversion is the double * -1.

Reply:
Aaaah yeah. I see, I did understand your logic to be honest (conversion from cartographic to lx200) I did not understand the previous conversion (the 360-) but since it was working before I had to ask :)